### ISSUE TITLE
Copy vote selection between multiple accounts

### ISSUE THREAD

**Issue by eospathmaker** [greymass/eos-voter#221]
Currently voting with multiple accounts does not have a way to copy your previous selections. This would be a useful feature.

**Comment by aaroncox** [comment]
Agreed - it'd be nice for even sharing your votes with other people!

One thing you could do, if they're all your own accounts, is use the vote proxy feature. Setup one account and register it as a proxy (under the tools section), then on all the other accounts, set their proxy to the first account with the votes. You'd then be able to control the voting of all of the accounts from a single account, and not need to copy/paste votes around!

**Comment by eospathmaker** [comment]
That is a workaround, but I would not want to create a connection between all of my accounts so not a good solution in all cases. Blockchain retains this connection forever and if you control accounts of others or are helping family members do their accounts, you might end up with the tax office claiming all of these are your accounts (just an example - not specific to my case).

**Comment by aaroncox** [comment]
Good point on that. 

We'll give it some thought and see if there's a good way to do it. My immediate thought is that we allow you to create an unsigned transaction, and then just sign + broadcast that same transaction for each account, effectively letting you copy/paste the actions. It'd be just a matter of making the UI intuitive enough to understand.

**Comment by eospathmaker** [comment]
That would do it :)
